packages/rollup.config.js

```diff
@@ -21,6 +21,7 @@ const checkImports = imports => {
     if (pattern instanceof RegExp) {
       return pattern;
     }
+    // If it's a string, turn it into a regex.
```

Comment:
:P i knew what the intention was, i just have no idea what the replacer regex does. is it just escaping any regex characters?